trie/sync.go

```diff
@@ -104,7 +105,8 @@ func (s *TrieSync) AddSubTrie(root common.Hash, depth int, parent common.Hash, c
 		return
 	}
 	key := root.Bytes()
-	blob, _ := s.database.Get(key)
+	panic(nil) // add position
```

Comment:
Is this intentional or a leftover from some debugging?

Reply:
Intentional, until trie syncing is updated for the new structure. Right now I wanted to immediately fail if anyone tried to use it.